presto-main/src/main/java/io/prestosql/sql/analyzer/Analysis.java

```diff
@@ -663,6 +669,32 @@ public boolean isOrderByRedundant(OrderBy orderBy)
         return redundantOrderBy.contains(NodeRef.of(orderBy));
     }
 
+    public boolean hasRowFilter(QualifiedObjectName table, String identity)
```

Comment:
s/identity/user

here and everywhere else

Reply:
No, because this is actually a SQL authorization identifier. Eventually, it will be either a user or a role, as expected by standard SQL authorization model.

Reply:
No, principal has way too much stuff. This is just an authorization identifier as described by the SQL spec.